zebrad/src/components/rpc.rs

```diff
@@ -0,0 +1,37 @@
+//! An RPC endpoint.
```

Comment:
Code Structure Question: Can we move the RPC server to `zebra_rpc::server`?

Then we can spawn it from `zebrad::commands::start` in `zebrad`, but run it by itself in the tests.

If we leave the server in `zebrad`, then tests will be a lot slower, because we will have to compile all of `zebrad` every time we change a test.

What do you think?

Reply:
https://github.com/ZcashFoundation/zebra/pull/3589/commits/f8d8986797c019ee99775051d2ce5a81c74e406c